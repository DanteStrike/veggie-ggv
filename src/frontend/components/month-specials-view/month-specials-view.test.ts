import MonthSpecialsView from "./month-specials-view";

it(`Month specials component should render correctly`, () => {
  const monthSpecials = [{
    title: `T1`,
    description: `D1`,
    img: `I1`,
    price: 1
  }, {
    title: `T2`,
    description: `D2`,
    img: `I2`,
    price: 2
  }];
  const monthSpecialsView = new MonthSpecialsView(monthSpecials);
  expect(monthSpecialsView.getElement()).toMatchSnapshot();
});
