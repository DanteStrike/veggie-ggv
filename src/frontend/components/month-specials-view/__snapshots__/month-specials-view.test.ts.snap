// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Month specials component should render correctly 1`] = `
<ul
  class="month-specials__offers-list"
>
  
                
  <li
    class="month-specials__offers-item clearfix"
  >
    
                
    <div
      class="month-specials__offer-img"
    >
      
                  
      <img
        alt="T1"
        height="248"
        src="assets/img/I1"
        width="248"
      />
      
                
    </div>
    
                
    <div
      class="month-specials__offer-wrap"
    >
      
                  
      <h3
        class="month-specials__offer-title"
      >
        T1
      </h3>
      
                  
      <p
        class="month-specials__offer-description"
      >
        D1
      </p>
      
                  
      <span
        class="month-specials__offer-price"
      >
        1$
      </span>
      
                
    </div>
    
              
  </li>
  <li
    class="month-specials__offers-item clearfix"
  >
    
                
    <div
      class="month-specials__offer-img"
    >
      
                  
      <img
        alt="T2"
        height="248"
        src="assets/img/I2"
        width="248"
      />
      
                
    </div>
    
                
    <div
      class="month-specials__offer-wrap"
    >
      
                  
      <h3
        class="month-specials__offer-title"
      >
        T2
      </h3>
      
                  
      <p
        class="month-specials__offer-description"
      >
        D2
      </p>
      
                  
      <span
        class="month-specials__offer-price"
      >
        2$
      </span>
      
                
    </div>
    
              
  </li>
  
            
</ul>
`;
